scoring/scoring_utils.py

```diff
@@ -190,9 +190,30 @@ def get_experiment_df(experiment_dir):
         data[column] = values
       trial_df = pd.DataFrame([data])
       workload_df = pd.concat([workload_df, trial_df], ignore_index=True)
-    num_trials = len(workload_df)
-    if num_trials != NUM_TRIALS:
-      warnings.warn(f'There should be {NUM_TRIALS} trials for workload '
-                    f'{workload} but there are only {num_trials}.')
     df = pd.concat([df, workload_df], ignore_index=True)
   return df
+
+
+## Get workload properties
+def get_workload_validation_target(workload):
+  """Returns workload target metric name and value.
+  """
+  workload_name = re.match(WORKLOAD_NAME_PATTERN, workload).group(1)
+  framework = re.match(WORKLOAD_NAME_PATTERN, workload).group(2)
+  workload_metadata = copy.copy(WORKLOADS[workload_name])
+
+  # Extend path according to framework.
+  workload_metadata['workload_path'] = os.path.join(
+      BASE_WORKLOADS_DIR,
+      workload_metadata['workload_path'] + f'{framework}',
+      'workload.py')
+  workload_init_kwargs = {}
+  print(workload_metadata['workload_path'])
```

Comment:
Could use `logging.info` instead of `print` and maybe add something like "workload path:".

Reply:
good catch, thanks!